CI/build-mudlet-for-windows.sh

```diff
@@ -90,10 +96,10 @@ fi
 export MUDLET_VERSION_BUILD="${MUDLET_VERSION_BUILD,,}"
 export BUILD_COMMIT="${BUILD_COMMIT,,}"
 
+# What is this needed for?
```

Comment:
This is used within the QMake project files to inform it of the locations of some library/header files - and on Windows it can be using the `cmd.exe` shell rather than a `bash` one for some of its internal shenanigans - so it is sensible to give it Windows "native" paths (`\` as the directory separator and, typically, `C:\` at the beginning of paths); as opposed to "MSYS2+Mingw-w64" paths (`/` and `/c/` {where needed for paths outside of the MSYS2+Mingw64 sub-system} respectively). The latter, in these scripts, being held by the `MINGW_INTERNAL_BASE_DIR` variable...